dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/messaging/BulkDataRequestEmailMessageSender.kt

```diff
@@ -0,0 +1,58 @@
+package org.dataland.datalandcommunitymanager.services.messaging
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.dataland.datalandcommunitymanager.model.dataRequest.BulkDataRequest
+import org.dataland.datalandmessagequeueutils.cloudevents.CloudEventMessageHandler
+import org.dataland.datalandmessagequeueutils.constants.ExchangeName
+import org.dataland.datalandmessagequeueutils.constants.MessageType
+import org.dataland.datalandmessagequeueutils.constants.RoutingKeyNames
+import org.dataland.datalandmessagequeueutils.messages.InternalEmailMessage
+import org.dataland.keycloakAdapter.auth.DatalandAuthentication
+import org.dataland.keycloakAdapter.auth.DatalandJwtAuthentication
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import java.util.*
+
+/**
+ * A class that manages generating emails messages for bulk and single data requests
+ */
+@Component
+class BulkDataRequestEmailMessageSender(
+    @Autowired private val cloudEventMessageHandler: CloudEventMessageHandler,
+    @Autowired private val objectMapper: ObjectMapper,
+) : DataRequestEmailMessageSenderBase() {
+    private val logger = LoggerFactory.getLogger(BulkDataRequestEmailMessageSender::class.java)
```

Comment:
Du könntest den logger über Autowiring in die Klasse injecten.

Siehe bspw. im `BulkDataRequestManager`.

Reply:
Ne, das w're ja eine anderer Logger. Nämlich dieser wrapper.

Reply:
Hmm. I see. Bin dran.

Reply:
Erledigt. Kein Logger mehr benötigt